reactMovieApp/src/api/getData.js

```
@@ -0,0 +1,14 @@
+export async function getData(url) {
+    let data = [];
+    
+    try{
+        let response = await fetch(url)
+        .then(response => response.json())
+        .then(data => { 
+            return data.results;
+        });
+        data = response;
+    } catch(err) { alert(err) }
```

please return the error here, and dont mix dom related functions with api